### PR TITLE
ci: run CI with GA with all braches after push event

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
**Description**

This PR enables running CI in Github Actions after push event.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches: [master]
```
Comment (mariajgrimaldi) [inline .github/workflows/ci.yml:5]: Concern: should we change this to '**'?

File: requirements/base.txt [hunk 2]
```diff
@@ -70,7 +70,7 @@ edx-rest-api-client==5.2.1  # via -r requirements/base.in, edx-ecommerce-worker
 enum34==1.1.10            # via cybersource-rest-client-python
 extras==1.0.0             # via cybersource-rest-client-python, python-subunit, testtools
 factory-boy==2.12.0       # via django-oscar
-faker==5.6.5              # via factory-boy
+faker==8.1.3              # via factory-boy
```
Comment (mariajgrimaldi) [inline requirements/base.txt:73]: This fixes:
https://github.com/eduNEXT/edunext-ecommerce/runs/3744167423?check_suite_focus=true

File: .github/workflows/ci.yml (hunk 1 above)
Comment (MoisesGSalas) [reply .github/workflows/ci.yml:5]: Probably, we might do some testing or exploring a new feature and having the tests directly on the branch would be easier than creating a PR that may not even be merged.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -4,7 +4,8 @@ on:
...
   pull_request:
-    branches: [master]
+    branches:
+      - '**'
 
 jobs:
   build:
```
Comment (MoisesGSalas) [inline .github/workflows/ci.yml:11]:
```suggestion
  build:
    name: '${{ matrix.testname }}'
```
The name I saw in the run is a bit difficult to parse. You can add the django-env or format it differently, but you get the gist.

Comment (mariajgrimaldi) [reply .github/workflows/ci.yml:11]: That would be neat. But I'm not sure it's worth creating a migration issue

Comment (MoisesGSalas) [reply .github/workflows/ci.yml:12]: Won't all this changes fall in to one issue?

**Comment by MoisesGSalas** [comment]
@mariajgrimaldi, the tests seem to be running twice, on push and on pull request. Do we expect PRs from external forks? maybe we can play around the [activity types](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request) and only run PRs that have a label? or simply not run the tests for PRs.

**Comment by mariajgrimaldi** [comment]
> @mariajgrimaldi, the tests seem to be running twice, on push and on pull request. Do we expect PRs from external forks? maybe we can play around the [activity types](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request) and only run PRs that have a label? or simply not run the tests for PRs.

Tests in PRs will run either way so I prefer removing the pull_request event.

**Review by MoisesGSalas — APPROVED** [review]

**Review by magajh — APPROVED** [review]

**Pull request merged**